packages/reverse-exports/tests/reverse-exports.test.ts

```diff
@@ -0,0 +1,251 @@
+import reversePackageExports, { _findPathRecursively } from '../src';
+
+describe('reverse exports', function () {
+  //   it('correctly reversed exports', function () {
+  //     // TODO figure out what the result should be if it doesn't match anything in exports
+  //     expect(reversePackageExports({ name: 'best-addon' }, './dist/_app_/components/face.js')).toBe(
```

Comment:
we need to keep this test i.e. a package with no exports

Reply:
Done.